js/src/setup-mc/setup-stepper/setup-accounts/faqs.js

```diff
@@ -69,11 +69,14 @@ const faqItems = [
 
 /**
  * @fires gla_setup_mc_faq
```

Comment:
💅  Missing id and action props.

💅  This could promote to a new refactor issue since `gla_setup_mc_faq` is kinda the same as `gla_get_started_faq` and other faqs. Could be interesting to do a refactor on this by using just `gla_faq` adding  `setup_mc` and `get_started` as props (context?)